types/src/block_metadata.rs

```diff
@@ -40,17 +44,49 @@ impl BlockMetaData {
 
     pub fn into_inner(
```

Comment:
what is inner here after the change?

Reply:
The inner here will be the arguments to the move value. To simplify the story we'll just pass in a bytearray so that we don't need to convert that map into a vector in Move.